sdk/core/azure-core/inc/azure/core/context.hpp

```diff
@@ -29,9 +29,9 @@ namespace Azure { namespace Core {
     /**
      * @brief Constructs an `OperationCancelledException` with message string as the description.
      *
-     * @param message The description for the exception.
+     * @param whatArg The explanatory string.
      */
-    explicit OperationCancelledException(std::string const& message) : std::runtime_error(message)
+    explicit OperationCancelledException(std::string const& whatArg) : std::runtime_error(whatArg)
```

Comment:
@JeffreyRichter and @CaseyCarter what do we do here where the suggested naming pattern of `what_arg` doesn't follow the SDK guideline? Should we add an exception in this case (no pun intended)?

Reply:
Just call the parameter 'what' without the Arg